mocha_test/forever_test.js

```diff
@@ -0,0 +1,44 @@
+var async = require('../lib/async');
```

Comment:
Thats what happening at the moment anyway. Would you want this named `forever.js`?